### PR TITLE
Update README.md

### PR DESCRIPTION
**Pull request by ajay-paratmandali**
Signed-off-by: Ajay Paratmandali <ajay.paratmandali@seagate.com>

## Problem Statement
<pre>
  <code>
  Story Ref (if any):
    Your Problem statement here...
         Update README for py-utils package
  </code>
</pre>
## Unit testing on RPM done
<pre>
  <code>
  Yes
  </code>
</pre>
## Problem Description
<pre>
  <code>
    Your Problem decription here...
         Getting confusion for understanding how py-utils work
  </code>
</pre>
## Solution
<pre>
  <code>
    Your Problem solution here...
         Add README.md
  </code>
</pre>
## Unit Test Cases
<pre>
  <code>
    Unit Testing details here...
Not needed
  </code>
</pre>

### CODE REVIEW

**Review by prathameshrodi — APPROVED** [review]

File: py-utils/README.md [hunk 1]
```diff
@@ -18,25 +18,65 @@ please email opensource@seagate.com or cortx-questions@seagate.com.
...
+```
+
+- RPM uninstall
+```
+$ yum remove cortx-py-utils
+```
+
+## Update new dependency package
+
+- Add package in `requirement.txt`.
```
Comment (mssawant) [inline py-utils/README.md:76]: s/requirement.txt./requirements.txt

Comment (mssawant) [inline py-utils/README.md:76]: s/requirement.txt/requirements.txt/

**Review by mssawant — APPROVED** [review]
Minor comment.

**Review by ajaysrivas — APPROVED** [review]

**Comment by ajaysrivas** [comment]
Please look at Codacy errors too.

File: py-utils/README.md (hunk 1 above)
Comment (ajay-paratmandali) [reply py-utils/README.md]: Done

Comment (ajay-paratmandali) [reply py-utils/README.md]: Done

**Comment by ajay-paratmandali** [comment]
`Please look at Codacy errors too.
@ajaysrivas 

Done
`

**Review by ajaysrivas — APPROVED** [review]

**Review by sachinpunadikar — APPROVED** [review]
+1

**Pull request merged**